conjur/config.py

```diff
@@ -9,14 +9,16 @@
 
 import logging
 import netrc
-import os.path
 
 from yaml import load, dump
 try:
```

Comment:
Why this try statement in imports? We dont know in advance version

Reply:
didn't add those in. This was before my time so I cannot speak to the intention (:

Reply:
okay

Reply:
Local libyaml is orders of magnitude faster than python native yaml lib. Here we just choose the fastest implementation possible.